app/assets/stylesheets/navigation.scss

```diff
@@ -64,7 +64,7 @@
 }
 
 .navbar-pf-vertical {
-  background: image-url('navbar.png');
+  background: #01749d url('navbar.png');
```

Comment:
can you please use the `$primary_color` (or similar) variable from `colors.scss` ?

Reply:
in theory, if the logo is small enough, you can encode it in css (to avoid the roundtrip) but in this particular case (that you want to be able to make it theamable, perhaps not?)